CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Documentation for hipCUB is available at
 
 ### Added
 * Add support for large indices in `hipcub::DeviceSegmentedReduce::*`. rocPRIM's backend provides support for all reduce variants, but CUB's does not have support yet for `DeviceSegmentedReduce::Arg*`, so large indices support has been excluded for these as well in hipCUB.
-
+* Add -t smoke option in rtest.py. It will run a subset of tests such that the total test time is in 5 minutes. Use python3 ./rtest.py --test smoke or python3 ./rtest.py --t smoke to execute smoke test.
```

Comment:
Just a little thing I noticed here in the documentation - I think that usually single letter options usually use a single dash. So maybe the final usage example on the second line should read:
"or python3 ./rtest.py **-t** smoke" (instead of "**--t** smoke").

Reply:
Hi, yes that would usually be the case, but for rtest.py you can use both --t or -t. I can still make the change for consistenc sake